config/locales/sufia.en.yml

```diff
@@ -47,6 +47,8 @@ en:
       local_ingest:
         tab_label: "Network/Server Location"
       processing: "File is being processed; you may edit when processing has completed"
+      permissions_message: "Updating file permissions. This may take a few minutes. You may want to refresh your browser or return to this record later to see the updated file permissions."
+      change_permissions_message: "<p>You have changed the permissions on this <%= curation_concern.human_readable_type %>, <i><%= curation_concern %></i>, making it visible to <b><%= visibility_badge(curation_concern.visibility) %></b>.</p><p>Would you like change all of the files within the <%= curation_concern.human_readable_type %> to <b><%= visibility_badge(curation_concern.visibility) %></b> as well?</p>"
```

Comment:
@scherztc here ois where you could change the key to change_permissions_message_html